tests/test_account.py

```diff
@@ -285,3 +285,24 @@ def test_view_account_delete_confirm(customer_user, authorized_client):
     assert response.status_code == 302
     customer_user = User.objects.filter(pk=customer_user.pk).first()
     assert customer_user is None
+
+
+def test_add_names_to_user(customer_user):
+    name_form = NameForm(
+        {'first_name': 'Jan', 'last_name': 'Nowak'},
+        instance=customer_user)
+    name_form.is_valid()
+    name_form.save()
+    updated_user = User.objects.get(pk=customer_user.pk)
+    assert updated_user.first_name == 'Jan'
+    assert updated_user.last_name == 'Nowak'
+
+
+def test_add_names_to_user_post(customer_user, authorized_client):
```

Comment:
I think we could change the convention to `test_view_functionname` or even `test_functionname`